packages/lib/src/components/Dropin/components/PaymentMethod/PaymentMethodItem.scss

```diff
@@ -55,15 +55,15 @@
     justify-content: space-between;
     font-weight: 400;
     font-size: $font-size-medium;
-    padding: 16px;
-    padding-left: 48px;
+    padding: $spacing-medium - $spacing-xsmall;
```

Comment:
Why not use `$spacing-medium-small` here? Goal is to use `12px`, right?

Reply:
Because in my opinion makes it more obvious why there is -4px, it's the padding of the button element.

Reply:
I understand. It makes sense from your perspective. But if someone works on this file at some point, he/she might wonder why this. If the goal is to not use `$spacing-medium-small` in order to make it more obvious the calculation being done here for the padding, you could do:

```
$button-padding: $spacing-xsmall;
padding: $spacing-medium - $button-padding;
```

It would make it more explicit why is like that.

Reply:
I do like this even more, I will update it to add that exactly that.